build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -186,9 +258,75 @@
         Overwrite="True"
     />
   </Target>
+
+  <!-- The condition below is to work around a bug in xbuild which attempts to batch
+       even if there are no _MonoCrossRuntime items and the Copy task fails
+  -->
+  <Target Name="_ConfigureCrossRuntimes"
```

Comment:
Should we keep naming/terminology consistent? In some places we're calling it "cross", e.g. `_ConfigureCrossRuntimes`, while in other places we're calling it "aot", e.g. `$(AndroidSupportedTargetAotAbis)`. Do "cross" and "aot" mean the same thing? Different things?

If they're the same thing, we should use the same noun, probably "Aot". If they're different things, it might be good to document that difference somewhere so we're all on the same page.